src/public/pages/examples/querystring-params.js

```diff
@@ -0,0 +1,63 @@
+import K, * as U from "karet.util"
+import * as L    from "partial.lenses"
+import React     from "karet"
+
+import Link from "../../components/link"
+
+import { TextInput }       from "../../components/text-input"
+import { PrettyStringify } from "../../components/pretty-stringify"
+
+const addQuerystringParams = params =>
+  params.modify( U.append( [ '', '' ]  ) )
+
+const QuerystringParam = ( { param } ) =>
+  <tr>
+    <td><TextInput value={ U.view( [ 0 ], param ) }/></td>
+    <td><TextInput value={ U.view( [ 1 ], param ) }/></td>
+    <td><button onClick={ () => param.remove() }>Remove</button></td>
+  </tr>
+
+const makeQuerystring =
+  U.ifElse( U.isNil
+          , _ => ''
+          , U.pipe( U.map( U.pipe( U.map( encodeURIComponent )
+                                 , U.join( '=' )
+                                 )
+                         )
+                  , U.join( '&' )
+                  , U.concat( '?' )
+                  )
+          )
+
+const paramsPairs = params =>
+  U.view( L.iso( U.toPairs, U.fromPairs ), params )
```

Comment:
The `params` parameter can be eliminated here as `U.view` is curried.

BTW, it can be advantageous to keep accessors like this as plain optics and call `U.view` only at point of where you have data (params) at hand.  The reason for that is that `U.view(lens, data)` returns a view of the `data`, which is not as composable as the `lens`.  In a context like this the additional composability isn't needed of course, but in the general case that is good thing to keep in mind.